### ISSUE TITLE
[ACCESSIBILITY] Settings screen - Improper Sequence once selecting dropdown item

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4181]
**Describe the issue**
When a screen reader user is gesturing through the settings page, upon selecting the dropdown (Not labeled properly is Bug # 4182) the user is taken back to the "Settings" title rather than the focus being taken to the first item in the dropdown list "Maps & Wayfinding".

**To Reproduce**
**Steps to reproduce the behavior:**
1. Using TalkBack
2. Go to 'Settings.'
3. Click on 'Contact .'

**Expected behavior**
The dropdown should be opened and the focus taken to the first item in list "Maps & Wayfinding".

**Actual behavior**
Once selecting dropdown, the user's focus is taken back to the title "Settings" which is confusing and causes unnecessary gesturing.

**Screenshots**
![20240625_185736](https://github.com/rokwire/illinois-app/assets/82892873/42cbce5f-1136-49df-82cd-502c9cd1f0f8)
![20240625_184941](https://github.com/rokwire/illinois-app/assets/82892873/b97be0c0-ec5c-4584-bb9d-081919d4cd37)

**Testing Environment & Devices (please complete the following information):**
•	OS: Android Version 13
•	Device: Google Pixel 4a

**Additional context**

**Comment by acovher2** [comment]
@pmarkhennessy This same behavior is on the Appointments screen Mckinley dropdown as well. Y'all should check anywhere in the app that dropdowns are used and ensure this focus sequence gets corrected.

**Comment by TodorBachvarov** [comment]
@acovher2 this should be fixed in version 6.0.18

**Comment by acovher2** [comment]
@TodorBachvarov Awesome!! Amazing turnaround time on this as well. I will retest as soon as I get that version. 

Kindly, Amanda

**Comment by acovher2** [comment]
Verified fixed in 6.0.32
